include/graph/Node.hpp

```diff
@@ -75,12 +75,12 @@ struct Node : APIObject<Node>, std::enable_shared_from_this<Node>
 		return typedNodes;
 	}
 
-	template<typename T, template<typename _> typename Container>
-	static typename T::Ptr getExactlyOne(const Container<Node::Ptr>& nodes)
+	template<typename T, template<typename _1, typename _2> typename Container>
+	static typename T::Ptr getExactlyOne(const Container<Node::Ptr, std::allocator<Node::Ptr>>& nodes)
 	{
 		std::vector<typename T::Ptr> typedNodes = Node::filter<T>(nodes);
 		if (typedNodes.size() != 1) {
-			auto msg = fmt::format("looked for {}, but found [{}]", name(typeid(T)), getNodeTypeNames(nodes));
+			auto msg = fmt::format(fmt::runtime("looked for {}, but found [{}]"), name(typeid(T)), getNodeTypeNames(nodes));
```

Comment:
> To pass a runtime format string wrap it in [fmt::runtime()](https://fmt.dev/latest/api.html#_CPPv4N3fmt7runtimeE11string_view).

But `"looked for {}, but found [{}]"` clearly is not a runtime string. So why `fmt::runtime()`?